projects/packages/search/src/class-stats.php

```diff
@@ -14,7 +14,8 @@
  * Search stats (e.g. post count, post type breakdown)
  */
 class Stats {
-	const CACHE_EXPIRY             = 5 * 60; // Five minutes in seconds.
+	const MINUTES_IN_SECONDS       = 60;
+	const CACHE_EXPIRY             = 5 * self::MINUTES_IN_SECONDS;
```

Comment:
Shall we use `MINUTE_IN_SECONDS` to match https://codex.wordpress.org/Easier_Expression_of_Time_Constants?

Reply:
Fixed!